atlasdb-console/src/main/groovy/com/palantir/atlasdb/console/AtlasConsoleBinder.java

```diff
@@ -99,7 +99,8 @@ public Object call() {
             System.out.println("\nTopics with help messages:\n"); // (authorized)
             System.out.println(help.keySet().toString()); // (authorized)
             System.out.println("\nCall help(\'topic\') to view information on a specific topic.\n"); // (authorized)
-            System.out.println("If you'd like to enable database mutations (not recommended), re-run console with the 'mutations_enabled' flag\n"); // (authorized)
+            System.out.println("If you'd like to enable database mutations (not recommended), re-run console with the"
+                    + " 'mutations_enabled' flag\n"); // (authorized)
```

Comment:
Is this OK? Lines 114-115 suggests that `(authorized)` should be indicated on both lines

Reply:
Had a look internally - this is probably OK (see e.g. the large internal product's class that inspects atlas's lock service)